src/app/Services/CommentService.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Services;
+
+use App\Models\Post;
+use App\Models\Comment;
+use App\Http\Requests\CommentRequest;
+use Illuminate\Database\Eloquent\Collection;
+
+class CommentService
+{
+    public function getCommentsForPost(Post $post): Collection
```

Comment:
Add comment!

Reply:
I have resolved the comment, sir! Thank you. 🙇